libraries/matchers.rb

```diff
@@ -0,0 +1,6 @@
+# chef-splunk/libraries/matchers.rb
+if defined?(ChefSpec)
```

Comment:
in general you don't want to define chefspec matchers anymore if you can avoid it. Every resource gets a matcher out of the box now.